src/main/java/com/amazon/opendistroforelasticsearch/security/authtoken/AuthToken.java

```diff
@@ -0,0 +1,46 @@
+package com.amazon.opendistroforelasticsearch.security.authtoken;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.time.Instant;
+
+public class AuthToken {
```

Comment:
This class is not fully written as a part of this CR. It will be initialized in the upcoming one. This class is just introduced as other class has dependency. 

public AuthToken getTokenByClaims(Map<String, Object> claims) {
        return null;
    }